D6StarWars/D6StarWars.css

```diff
@@ -337,81 +385,438 @@ input.sheet-options-flag.sheet-main-options + span {
     display: block;
 }
 
-input.sheet-options-flag.sheet-main-options:hover + span {
+.charsheet input.options-flag.main-options:hover + span {
     opacity: 1;
 }
 
-input.sheet-options-flag:checked + span {
+.charsheet input.options-flag:checked + span {
     color: white;
     display: block;
 }
 
-input.sheet-selector-switch:checked ~ .sheet-master-container .sheet-content .sheet-options,
-input.sheet-selector-switch:checked ~ .sheet-master-container .sheet-content .sheet-options-flag:checked ~ .sheet-display  {
+.charsheet input.selector-switch:checked ~ .master-container .content .options,
+.charsheet input.selector-switch:checked ~ .master-container .content .options-flag:checked ~ .display  {
     display: none;
 }
 
-input.sheet-selector-switch:checked ~ .sheet-master-container .sheet-content .sheet-options-flag:checked ~ .sheet-options {
+.charsheet input.selector-switch:checked ~ .master-container .content .options-flag:checked ~ .options {
     display: block;
 }
 
+.charsheet div.sheettype-title{
+    position:absolute;
+    right: 5px;
+    top:-35px;
+}
 /*-------------------------------------------*/
 /*--------------- PC Section-----------------*/
 
-.charsheet .sheet-physical-params {
+.charsheet .physical-params {
     display: flex;
     margin-top: 25px;
 }
 
-.charsheet .sheet-physical-params label {
+.charsheet .physical-params label {
     text-align: center;
 }
 
-.charsheet .sheet-physical-params input {
+.charsheet .physical-params input {
     display: block;
     width: 100% !important;
+    text-align: center;
 }
 
-.charsheet input[type=number].sheet-smallshortnumber {
-    width: 60px;
-    text-align: right;
-}
-
-.charsheet .sheet-custom-roll-table {
+.charsheet .custom-roll-table {
     display: table;
     width: 100%;
 }
 
-.charsheet .sheet-custom-roll-table button.sheet-d6-dice {
+.charsheet .custom-roll-table button.d6-dice {
     width: 35px;
     height: 18px;
 }
+.charsheet textarea.description{
+    height:6em;
+    width:97%
+}
+.charsheet textarea.description-phys{
+    height: 6em;
+    width: 97%;
+    margin-top: 3px;
+}
+.charsheet textarea.specabilities{
+    width: 95%;
+    height: 12em;
+}
+
+.charsheet th.init{
+    vertical-align: top;
+}
+.charsheet td.init{
+    padding-left: 6px;
```

Comment:
As I said. Every single one. Note that this also renders the sheets incompatible with normal roll20.